cmd/meroxa/root/apps/deploy.go

```diff
@@ -990,11 +980,13 @@ func (d *Deploy) Execute(ctx context.Context) error {
 	}
 
 	d.specVersion = d.flags.Spec
+
+	// Make the latest version default for Ruby (Go it's 0.2.0 ready, but not making it default for now)
 	if d.specVersion == "" && d.lang == ir.Ruby {
```

Comment:
I think we should do the same for go, because there's so many cli changes, that would not work without core

Reply:
@dianadoherty yup! changed that!